### DISCUSSION TITLE
authenticatedItem returns null on fresh Keystone App

### DISCUSSION THREAD

**Discussion by wchorski** [keystonejs/keystone#8442] [Questions] [answered]
Is authorization and session data supposed to work out of the box on a fresh Keystone App? 

With a fresh app with one user, I login and see the `keystonejs-session` in my cookies. 

I go to the Apollo Sandbox to make a query

## Query
```json
query User {
  authenticatedItem {
    ... on User {
      email
      id
      
    }
  }
}
```

## Return
```json
{
  "data": {
    "authenticatedItem": null
  }
}
```

Is this a problem with **SQLite**? Do I need to tweak settings in the `auth.ts`?

**Comment by arsekil** [comment]
I don't know how is this supposed to be working, but I'm also interested in the answer.

**Comment by marekryb** [comment, marked as answer]
It used to work out of the box with the old Apollo Playground.
With the new Apollo Sandbox that is included in Apollo Server 4, cookies are not sent with request by default.

You can enable it by opening settings
![apollo1](https://user-images.githubusercontent.com/60657309/229014529-03db703c-bc8e-4427-ab44-a39ab2c90ea5.png)
and then switching this:
![apollo2](https://user-images.githubusercontent.com/60657309/229014660-13afa0a0-5fa4-4a58-bea1-730cdcec232d.png)

It works in simple project, but probably gets complicated when your server has its own CORS settings and such (not sure).

Anyway what always work is adding HTTP header to each request (you can set shared one for all tabs) - ``"Authorization": "Bearer XXXXXX"`` where XXXXX is ``sessionToken`` from ``authneticateUserWithPassword`` mutation.
